JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/definitions/JavaExpansionMemberVisitor.java

```diff
@@ -244,13 +244,20 @@ public Void visitCaster(CasterMember member) {
 		final JavaWriter methodWriter = new JavaWriter(context.logger, member.position, writer, javaMethod, member.definition, true);
 
 		methodWriter.label(methodStart);
-		methodWriter.nameVariable(0, "expandedObj", methodStart, methodEnd, context.getType(this.expandedClass));
-		methodWriter.nameParameter(0, "expandedObj");
 
-		int i = 1;
+		int i = 0;
```

Comment:
💭 Thought: We probably need to do similar changes to the other methods in this class, but so far only this one caused Tests to fail 🤷🏽‍♂️